pkg/operator/controller/testdata/default_corefile_no_cache_configured

```diff
@@ -0,0 +1,26 @@
+.:5353 {
+    bufsize 512
+    errors
+    log . {
+        class error
+    }
+    health {
+        lameduck 20s
+    }
+    ready
+    kubernetes cluster.local in-addr.arpa ip6.arpa {
+        pods insecure
+        fallthrough in-addr.arpa ip6.arpa
+    }
+    prometheus 127.0.0.1:9153
+    forward . /etc/resolv.conf {
+        policy sequential
+    }
+    cache 900 {
+        denial 9984 30
+    }
```

Comment:
For a test of "no cache configured", could you remove this stanza entirely?

Reply:
If I understand correctly, you're suggesting that "no cache configured" means "no cache stanza". In our case, there's always a cache stanza because if the API isn't configured then the default values are used as opposed to not having cache at all. The values we ship with are also different than the CoreDNS defaults so as far as I know, we wouldn't hit a case where there'd be a Corefile with no cache stanza.